app/classes/document_query.rb

```diff
@@ -14,7 +12,7 @@ class DocumentQuery
 
   def initialize(options)
     @options = options
-    @language = options[:language]
+    @language = options[:language] || 'en'
```

Comment:
@noremmie , because we are no longer populating the raw fields without the language suffix, searches will not work without the language parameter. However, we shouldn't need to support searches without a language, as documents created without a language default to 'en':
https://github.com/GSA/i14y/blob/master/app/controllers/api/v1/documents.rb#L84

Reply:
One potential downside to this is that when someone searches an i14y drawer from the admin center, the admin center doesn't send the language field as part of the search request to i14y. If a customer has indexed a bunch of non-English content, then they won't be able to search it via the admin center once the locale for such searches defaults to English.

This feels like an acceptable downside given how few i14y sites have non-English content indexed. Maybe the solution is to just add a new story that lets customers select the locale they want to search with from the admin center. So I'm OK with this change as long as you are. What do you think?

Reply:
@noremmie , that's a good point. However, as each affiliate's `language` value is passed for any of their I14y searches, I think the better approach is to pass that value for drawer searches in the admin center. An affiliate could have `es` docs in their drawer, but those would only be searchable if the affiliate language was also `es`.

Reply:
That does indeed sound like a better approach. Sounds good!

Reply:
@noremmie , it occurred to me that there could be multi-lingual drawers shared between affiliates using each of those languages. I ran a script to find multi-lingual drawers in production, and there were only a few:
```
------production-i14y-documents-da_test-v1------
en: 75
es: 4
------production-i14y-documents-nhlbi_drupal-v1------
en: 14701
es: 198
------production-i14y-documents-nps_site-v1------
en: 126043
es: 568
fr: 38
ru: 14
de: 11
------production-i14y-documents-fda_dev-v1------
en: 71
es: 1
```

The good news is that none of those are shared by affiliates with varying locale settings, so there shouldn't be confusion if different affiliates sharing the same drawer show different documents in their drawer search from the admin center. Also, both `nhlbi` and `nps` are moving to Garfield, while the other two drawers look like test drawers.